.github/workflows/update-deps.yml

```diff
@@ -0,0 +1,37 @@
+name: Update Dependencies
+
+on:
+  # Run every day.
+  schedule:
+    - cron: '0 3 * * *'
+  # And on on every PR merge so we get the updated dependencies ASAP, and to make sure the changelog doesn't conflict.
+  push:
+    branches:
+      -master
+
+jobs:
+  update-cli:
+    uses: denrase/github-workflows/.github/workflows/updater.yml@9f98d2bca80719220d03f18740c052b9ae6ebbaa # TODO: Use sentry workflow when merged.
+    with:
+      path: script/sentry-cli.properties
+      name: CLI
+    secrets:
+      api_token: ${{ secrets.CI_DEPLOY_KEY }}
+  update-binaries:
+    runs-on: macos-latest
```

Comment:
btw ubuntu runners are more readily available and faster to start up, most of the times.